<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IM Client - Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_server">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QFormLayout" name="formLayout_server">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ip">
        <property name="text">
         <string>Server IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ip"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_port"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_user">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QFormLayout" name="formLayout_user">
      <item row="0" column="0">
       <widget class="QLabel" name="label_clientId">
        <property name="text">
         <string>Client ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_clientId"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nickname">
        <property name="text">
         <string>Nickname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_nickname"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_connect">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="minimumHeight">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Status: Disconnected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
